keyboards/pabile/p40/keymaps/ortho/config.h

```diff
@@ -0,0 +1,18 @@
+
+#include "../../config.h"

```

Comment:
This isn't needed, as it's automatically included. 
```suggestion
#pragma once 
```